apps/website/src/pages/docs/components/forms/field.mdx

```diff
@@ -68,7 +68,7 @@ Any other type will fallback to a regular HTML `input` element. Eg: `email` will
 ```jsx
 <Form>
   <Field type="number" name="amount" label="Amount" value="10" />
-  {/* or: <NumberField name="amount" label="Amount" /> */}
+  {/* or: <NumberInputField type='number' name="amount" label="Amount" /> */}
```

Comment:
I'd assume the need to add the `type='number'` prop is actually a bug, but it's required to make the component work currently.